src/Illuminate/Support/Str.php

```diff
@@ -779,4 +779,85 @@ public static function createUuidsNormally()
     {
         static::$uuidFactory = null;
     }
+
+    /**
+     * Clean a string with easy to remember options
+     *
+     * @param $target
+     * @param array $options
+     *
+     * options include letters, numbers, space, comma, period, dash
```

Comment:
invalid syntax

Reply:
I think the options line was the error here - I removed it from the DocBlock.  I hope that's the only piece here?

Thanks Graham,